docs/sources/mimir/manage/tools/mimirtool.md

```diff
@@ -174,6 +174,19 @@ The following command verifies if alerts in an Alertmanager cluster are deduplic
 mimirtool alerts verify
 ```
 
+#### Render template
+
+You can render your alertmanager template by using the subcommand `render`.
+
+The following command render a template and prints it to the terminal.
+
+```bash
+# assuming we have written the templates out to the folder templates
+mimirtool alertmanager render --template-glob "templates/*" --template-data "alert_data1.json" --template-text '{{ template "my_message" . }}'
+```
```

Comment:
I've moved the comment into the lead-in text.
I've also fixed the quoting because the shell will expand the glob in `--template-glob "templates/*"` before it is passed to the command.

````suggestion
```bash
mimirtool alertmanager render --template-glob 'templates/*' --template-data alert_data1.json --template-text '{{ template "my_message" . }}'
```
````